### PR TITLE
Fixes sneaky powercreeper hiding under floor tiles

### PR DESCRIPTION
**Pull request by jknpj**
Closes #24863

:cl:
 * bugfix: Powercreeper will not hide under floor tiles anymore.

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
Powercreeper? Aww man

File: code/modules/power/cable.dm [hunk 1]
```diff
@@ -123,6 +122,10 @@ By design, d1 is the smallest direction and d2 is the highest
 	attached = null
 	..()								// then go ahead and delete the cable
 
+/obj/structure/cable/proc/reset_plane() //Set cables to the proper plane. They should NOT be on another plane outside of mapping preview
+	if(plane != ABOVE_PLATING_PLANE)
```
Comment (Exxion) [inline code/modules/power/cable.dm]: This seems completely pointless

Comment (jknpj) [reply code/modules/power/cable.dm]: I guess

**Comment by Probe1** [comment]
Powercreeper needs a rebalance.

**Comment by ghost** [comment]
What the fuck. Those vines were a cable subtype ? Jeez

**Pull request merged**